### DISCUSSION TITLE
Part Time Artist Contracts

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1842] [Art]
Hey artists 👋 

We've come a long way from where we started. Thanks to all of the community contributors we were able to bootstrap the art of this amazing game and create an awesome community of artists.

While we have 5 full-time engineers assisting with the vision of the game's technical architecture and assisting open source developers, we feel we haven't given enough love to the artists contributing to the project.

As we've scaled it has become harder to support our community of growing artists. There are a large amount of art projects to manage and quite honestly the team feels overwhelmed. To help us manage and flourish, we are opening some contract roles for members in the community to step up and help us improve our processes.

**Difference between a community artists and a contracted artist?**

Community artists are still relied on to contribute art for all aspects of the game - Bumpkins, decorations, maps, promo videos, fan art + rare SFTs.

A contracted artist is expected to contribute art, but has additional expectations such as:

- Improving Guidelines
- Providing Feedback on Github and Discord (or live pixel art sessions)
- Surfacing the noteworthy community contributions
- Checking for plagiarism
- Managing design briefs (i.e. create a theme for an island)

The last thing we want is to create a split between contracted & non-contracted artists. The one's that are contracted are to act as 'servants' to the community and help mentor and improve our standards ❤️ 

We are going to be on the lookout for additional artists to join this team and also for commissioned pieces of work.

While we only have 2 roles open at the moment, we plan to scale and grow this to a fully functioning team 💪 

**Details**

1. This is a part time role, it is expected to contribute `3-4 hours of your time per day`
2. $100 USD weekly for the first trial month. Following that, $150 USD weekly. This isn't an insane amount in Western countries, but we are hoping it can greatly help contributors in other locations.
3. 3 month contract - flexible if you need time off for holidays, exams, study etc. 

**Position 1 - Bumpkin Designer**

_Start: January_

This role does include submitting Bumpkin designs but the main focus is to assist in coming up with Bumpkin ideas to share with the community, highlighting the community contributions, provide feedback and help artists with guidelines.

1. Collaborate with the core artists, plan and ideate upcoming ideas for Bumpkins
2. Provide feedback in Discord and Github on Bumpkin designs - ensure they meet our consistent guidelines + check for plagiarism across submissions
3. Design Bumpkins needed for team deadlines (partnerships, secret launches etc.) - 1 Bumpkin per week
4. Highlight the favourite community designs each week in Discord and help the team integrate them into the game
5. Clean up, resize + minor adjustments on existing Bumpkin designs.

_Weekly Deliverables_

- 1 Bumpkin design each week (including iterations/feedback) from core team
- Summary of best Community Bumpkin designs & best community ideas

**Position 2 - Level & Decoration Designer**

_Start: February_

A level and decoration designer's role is to work with the community to design level's & decorations/buffs to place on player's lands. This position is more focussed on creating a theme for an island, suggesting ideas, ensuring consistency & working with the community to produce designs and collectibles.

1. Collaborate with the team to set the vision for the islands and decorations
2. Provide feedback in Discord and Github on decorations - ensure they meet our consistent guidelines + check for plagiarism across submissions
3. Design in game decorations and craft-able items - 1-2 designs a week
4. Work with the community to create sprite sheet packs and themed assets for unique islands (e.g. Treasure Island, Stone Haven, Snow Kingdom)
5. Highlight the favourite community designs each week in Discord and help the team integrate them into the game
6. Clean up, resize + minor adjustments on existing Bumpkin designs.

_Weekly Deliverables_

- 1-2 self-drawn designs as per team's requirements
- Collection of community designed assets + ideas to promote

If you would like to hear more details or apply for this position, please fill in the form below ⭐ 

https://forms.gle/1Q1kjYUEM3AdjzoH7

**Comment by NycksonV** [comment]
I want to apply
I would like to know more about this announcement and how I can participate.
I'm user Nyck#4168 on Discord

**Comment by Netherzapdos** [comment]
Happy to see designers have this opportunity 😄 Not 100% sure yet but i'd like to hear some details first 😋

**Comment by JcEii** [comment]
I'm willing to apply and fully prepared on what will be the details to be qualified... 🙆‍♂️

**Comment by Vergelsxtn** [comment]
More details please😁

**Comment by LittleEins** [comment]
Interested!

**Comment by On1el** [comment]
Wow, this is some serious competition.
I'll admit it! I've been dreaming of making the team)
Is it time to take the trump cards out of our sleeves?
![1531117694](https://user-images.githubusercontent.com/104461938/206644209-a14923a0-1883-4db9-9a94-a10180ee938b.png)

**Comment by digital-farms** [comment]
I'm ready.
I want to join the ranks of SFL artists!
Ready to work every day, my work you probably saw) Will be very happy!

**Comment by paluras** [comment]
Up for the challenge!

**Comment by sfwhat** [comment]
I'm sure that the team will decide a fitting artist🤘 I would be more comfortable having assignments from whomever gets the job for now as I got certified most recently.

**Comment by Synedes** [comment]
Lovely to see this happen, good luck to all concerned

**Comment by Rylinn** [comment]
Will be future positions posted here? I am interested on Position 2 as Level & Decoration Designer.
